jac-uk/admin
# #1912 Results should use z-score

## What's included?
Updates results pages to use Average Z Score, for qualifying tests, instead of Average Percentage
Closes #1912 

## Who should test?
✅ Product owner
✅ Developers

## How to test?
Set up or select an exercise which has qualifying tests (CAT & SJT). Run each test and in admin process the results.

1. Notice that for each test the results are ranked by percentage, e.g.
<img width="409" alt="image" src="https://user-images.githubusercontent.com/8524401/223483861-3fe1a67b-da15-43ba-8c24-0a4f7d772f71.png">

When a pass mark has been chosen and results completed for both tests you will be able to process the combined CA + SJ scores.

2. Notice that CA + SJ scores are ranked according to average z score
<img width="326" alt="image" src="https://user-images.githubusercontent.com/8524401/223484023-5c9cdc10-99ba-400f-80a8-8d88b5c630a1.png">

You should be able to set a pass mark and complete the results for 'CA + SJ scoring'.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
